model/src/pyrenew/model/hospitalizations.py

```diff
@@ -59,6 +80,10 @@ def __init__(
         Returns
         -------
         None
+
+        Notes
+        -----
+        TODO: See Also
```

Comment:
Best to avoid `TODO` in code, especially code merged to `main`. If this needs to be merged with this not done, we should make an issue.